index-html/about.html

```diff
@@ -0,0 +1,152 @@
+              </a>
+            </li>
+          </ul>
+        </div>
+      </nav>
+    </header>
+
+    <!-- Hero section -->
+    <section class="hero-about" id="home">
+      <div class="nav-toggle">
+        <div class="burger bar1"></div>
+        <div class="burger bar2"></div>
+        <div class="burger bar3"></div>
+      </div>
+      <nav class="mobile-nav">
+        <ul class="nav-list">
+          <li class="nav-item">
+            <a href="index.html" class="nav-link">Home</a>
+          </li>
+          <li class="nav-item">
+            <a href="about.html" class="nav-link">About</a>
+          </li>
+          <li class="nav-item">
+            <a href="#program" class="nav-link">Program</a>
+          </li>
+          <li class="nav-item">
+            <a href="#speakers" class="nav-link">Speakers</a>
+          </li>
+        </ul>
+        <a href="">
+          <button class="nav-btn btn">CC Campaign</button>
+        </a>
+      </nav>
+      <div class="hero-body">
+        <img
+          class="about-bg-img"
+          src="./images/about_title.png"
+          alt="about hero background"
+        />
+        <div class="about-hero-underline"></div>
+        <p class="hero-subtitle">
+          The Education Enclave, organized biennially, serves as a hub where
+          advocates of open education from diverse backgrounds, including
+          educators, researchers, and activists, converge. This event brings
+          together individuals committed to fostering open culture and those who
+          actively promote and participate in the field of education.
+        </p>
+        <p class="contact">
+          Please contact us via email for any further question about CC Global
+          Submit 2015! <br /><a href="#">ccsummit@gmail.com </a>
+        </p>
+      </div>
+    </section>
+    <hr />
+    <section class="about-section">
+      <h2 class="about-title">Creative Commons Global Submit 2015 logo</h2>
+      <div class="about-underline"></div>
+      <div class="paragraph-container">
+        <p class="about-subtitle">
+          The winner of the logo contest, which was held from June 8th to July
+          7th, was decided through online voting and evaluation by the judges.
+          After consultation with the winning designer, the logo of CC Global
+          Summit 2015, which was optimized for use on the website and
+          promotional materials, is as follows.
+        </p>
+        <img class="ccsubmit-logo" src="./images/about-logo.png" alt="" />
+      </div>
+    </section>
+    <hr />
+    <section class="previous-submit">
+      <h2 class="ps-title">See the past CC Global Submit</h2>
+      <div class="about-underline"></div>
+      <p class="ps-subtitle">Take a look at the last two CC Global Submit.</p>
+      <div class="previous-images">
+        <img src="./images/about_2011 1.png" alt="image" class="img2011" />
+        <img src="./images/about_2013.png" alt="image" />
```

Comment:
- [ ] Kindly consider adding these cards manually not as images this will showcase your skills in the usage of images and backgrounds to enhance the look of a website. as this is among the learning objectives of this project

Reply:
done.